tslint.json

```diff
@@ -23,6 +23,7 @@
       "allow-pascal-case",
       "allow-leading-underscore"
     ],
-    "semicolon": [true, "always"]
+    "semicolon": [true, "always"],
+    "no-unused-variable": true
```

Comment:
Are we able to comment in json config files? If so, it might be nice to clarify why "no unused variables" is configured in both tslint and tsconfig.

Reply:
Unfortunately `vscode` complains that `Comments are not permitted in Json.`